langserver/symbol.go

```diff
@@ -259,7 +259,7 @@ func toSym(name string, bpkg *build.Package, recv string, kind lsp.SymbolKind, f
 		SymbolInformation: lsp.SymbolInformation{
 			Name:          name,
 			Kind:          kind,
-			Location:      goRangeToLSPLocation(fs, pos, pos+token.Pos(len(name))-1),
+			Location:      goRangeToLSPLocation(fs, pos, pos+token.Pos(len(name))),
```

Comment:
I am surprised that this doesn't require updating tests... any clue why that is?